src/ThermalScattering.tex

```diff
@@ -1,45 +1,122 @@
-\begin{table} \centering
+\begin{table}[H] \centering
+  \caption{\JXS\ array element definitions for thermal scattering \ACE\ Table.}
+  \label{tab:JXSThermalScattering}
   \begin{tabular}{rll}
     \toprule
-    Element  & Name   & Location Description \\
+    Element & Name   & Location Description                 \\
     \midrule
-    1        & ITIE   & Inelastic energy table \\
-    2        & ITIX   & Inelastic cross sections \\
-    3        & ITXE   & Inelastic energy/angle distributions \\
-    4        & ITCX   & Elastic cross sections \\
-    5        & ITCA   & Elastic angular distributions \\
-             & \ldots & \\
-    32       & ---    & \\
+    1       & ITIE   & Inelastic energy table               \\
+    2       & ITIX   & Inelastic cross sections             \\
+    3       & ITXE   & Inelastic energy/angle distributions \\
+    4       & ITCE   & Elastic energy table                 \\
+    5       & ITCX   & Elastic cross sections               \\
+    6       & ITCA   & Elastic angular distributions        \\
+            & \ldots &                                      \\
+    32      & ---    &                                      \\
     \bottomrule
   \end{tabular}
-  \caption{\JXS\ array element definitions for thermal scattering \ACE\ Table.}
-  \label{tab:JXSThermalScattering}
 \end{table}
 
+\subsection{Format of Individual Data Blocks}
+\subsubsection{\textsf{ITIE} Block}\label{sec:ITIEBlock}
+
+The format of the \Block{ITIE} is given in \Tableref{tab:ITIEBlock}. The index at the start of the block is \startblock{ITIE}=\jxs{1}. Note that \jxs{2}=\jxs{1}+1+$N_{in}$. Linear-linear interpolation is assumed between adjacent energies.
+
+\begin{BlockTable}{ITIE}
+  \startblock{ITIE}            & $N_{in}$                           & Number of inelastic energies \\
+  \startblock{ITIE}+1          & $E_{in}(l),l=1,\ldots,N_{in}$      & Energies                     \\
+  \startblock{ITIE}+1+$N_{in}$ & $\sigma_{in}(l),l=1,\ldots,N_{in}$ & Inelastic cross sections
+  \label{tab:ITIEBlock}
+\end{BlockTable}
+
+\subsubsection{\textsf{ITCE} Block}\label{sec:ITCEBlock}
+
+The format of the \Block{ITCE} is given in \Tableref{tab:ITCEBlock}. The index at the start of the block is \startblock{ITCE}=\jxs{4}.
+
+\begin{BlockTable}{ITCE}
+  \startblock{ITCE}            & $N_{el}$                      & Number of elastic energies \\
+  \startblock{ITCE}+1          & $E_{el}(l),l=1,\ldots,N_{el}$ & Energies                   \\
+  \startblock{ITCE}+1+$N_{el}$ & $P(l),l=1,\ldots,N_{el}$      & (See below)
+  \label{tab:ITCEBlock}
+\end{BlockTable}
+
+For incoherent elastic scattering (\nxs{5}$\neq4$), $\sigma_{el}(E_{el}(l)) = P(l)$ with linear-linear interpolation between points. For coherent elastic scattering (\nxs{5}$=4$), $\sigma_{el}(E)=P(l)/E$ for $E_{el}(l) \le E < E_{el}(l+1)$. Note that in this case, $\sigma_{el}(E)=0$ below $E_{el}(1)$. However, above $E_{el}(N_{el})$, $\sigma_{el}(E) = P(N_{el})/E$.
```

Comment:
I definitely like these separated out as individual equations! As for the relation between P(l) and sigma_el, I was following what the MCNP5 manual did for the ITCE block (which was to say sigma_el = E/P). From a Monte Carlo code developer perspective, that's actually what you want -- how do I calculate a cross section given this value (anyone familiar with the format for coherent elastic scattering based on ENDF-102 will immediately understand the relation too). From a technical manual standpoint, it might make more sense to define P(l) as you've done in the suggestion. I'm fine either way -- let me know what your decision is.

Reply:
I think it’s more clear to keep P(l) on the left side of the equation as that is what the table says is coming. The alternative would be more work as we’d have to refactor the table to say that the cross sections were coming. I don’t think that will be as helpful.

Reply:
I ended up keeping the equation with P(l) on the left side but added another sentence/equation that equations that the energies are Bragg edges and the xs is found by inverting the other equation. I personally like it a lot now but happy to make any further changes you'd like.

Reply:
I like what you've settled on. I think it's well explained.